bot/cogs/commands.py

```diff
@@ -101,27 +115,112 @@ async def on_message(self, message):
                 return
 
     @commands.command()
-    async def version(self, ctx):
+    async def version(self, ctx: commands.Context):
+        """Usage: `version`
+        Response: Fred's current version
+        Notes: Command is a useful is-alive check"""
         await self.bot.reply_to_msg(ctx.message, self.bot.version)
 
-    @commands.command()
-    async def help(self, ctx):
-        await self.bot.reply_to_msg(ctx.message, "Sorry, this command is temporarily unavailable")
+    @commands.group()
+    async def help(self, ctx: commands.Context) -> None:
+        """[Help Commands!](https://www.youtube.com/watch?v=2Q_ZzBGPdqE)
+        Usage: `help [commands/crash(es)/special/media_only/webhooks] [page: int/name: str]`
+        Reply: information about what you requested"""
+        if ctx.invoked_subcommand is None:
+            await self.help_special(ctx, name='help')
+            return
+
+    async def _send_help(self, ctx: commands.Context, **kwargs):
+        if not await self.bot.checked_DM(ctx.author, **kwargs):
+            await ctx.reply("Help commands only work in DMs to avoid clutter. "
+                            "You have either disabled server DMs or indicated that you do not wish for Fred to DM you."
+                            "Please enable both of these if you want to receive messages.")
```

Comment:
Send a DM even if they've asked not to get DMs. That option is for impromptu DMs, requested DMs are fine. You can replace this with a message to ask them if we can DM them if you want. Ideally an ephemeral message, there has to be a way to use them